cluster/manager/pair.go

```diff
@@ -197,12 +197,20 @@ func (c *ClusterManager) GetPair(
 	if id == "" {
 		id, err = getDefaultPairId()
 		if err != nil {
-			return nil, err
+			if err == kvdb.ErrNotFound {
+				return nil, fmt.Errorf("No default cluster pair id set.")
```

Comment:
we could take out "id" ? i mean just say "No default cluster pair found"

Reply:
K will do.   Thx